src/main/java/com/neprozorro/exceptions/InvalidInputFormatException.java

```diff
@@ -0,0 +1,14 @@
+package com.neprozorro.exceptions;
+
+import java.math.BigDecimal;
+
+public class InvalidInputFormatException extends RuntimeException{
+
+    public InvalidInputFormatException(){
+        super("Invalid input format. You have to paste two values. Paste min and max total price");
+    }
+
+    public InvalidInputFormatException(BigDecimal minValue, BigDecimal maxValue){
+         super("Invalid input format. Second value: " + maxValue + " less then first value: " + minValue);
```

Comment:
In my view, the description has to sound like this:
"Not expected value. Expected format /"<min Price>,<max Price>/", where <min Price> (" + minPrice + ") can`t be less than the <maxPrice>( " + maxPrice + "). For ignoring <max Price>,  instead <max Price> put 0."

Reply:
Agree with @YuriiZaripa it's better to provide as much information as possible. 
Also use String.format() instead and provide values separately instead of using concatenation.